linux_docker_resources/Dockerfile

```
@@ -89,18 +89,23 @@ RUN if test ${PLATFORM} = x86; then apt-get update && apt-get install --no-insta
 RUN apt-get update && apt-get install --no-install-recommends -y python3-pexpect
 
 # Get and install the RTI web binaries.
-# Connext 5.3.1 for focal and earlier.
-RUN if test ${UBUNTU_DISTRO} != jammy; then cd /tmp && curl --silent https://s3.amazonaws.com/RTI/Bundles/5.3.1/Evaluation/rti_connext_dds_secure-5.3.1-eval-x64Linux3gcc5.4.0.tar.gz | tar -xz; fi
-RUN if test ${UBUNTU_DISTRO} != jammy; then cd /tmp && tar -xvf /tmp/openssl-1.0.2n-target-x64Linux3gcc5.4.0.tar.gz; fi
+# Connext 5.3.1 for Galactic and earlier.
+RUN if [ "$ROS_DISTRO" != "galactic" -o "$ROS_DISTRO" != "foxy" ]; then \
```

Isn't this logic inverted?  Shouldn't this be `$ROS_DISTRO = "foxy" -o "$ROS_DISTRO = "galactic"` ?

---

You are correct. These are what happens when you write conditionals late at night.
```suggestion
RUN if [ "$ROS_DISTRO" = "galactic" -o "$ROS_DISTRO" = "foxy" ]; then \
```